Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1012,3 +1012,111 @@ struct WebCore::NavigationPreloadState {
     String headerValue;
 };
 #endif // ENABLE(SERVICE_WORKER)
+
+enum class WebCore::RenderingMode : bool
+
+enum class WebCore::RenderingPurpose : uint8_t {
+    Unspecified,
```

Comment:
it seems very unfortunate that we are re-separating the enums from their sadly-necessary duplication; it was so nice that we had gotten to a place where they could be in the same file...

Reply:
Once we have compilers that support the reflexpr keyword, we can remove all of them.